types/cheesy-api/cheesy-api-tests.ts

```diff
@@ -0,0 +1,4 @@
+const obj: Subject = {
+	name: 'Someone',
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/cheesy-api/cheesy-api-tests.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.